src/test/java/io/anserini/collection/TwentynewsgroupsCollectiontest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Anserini: A Lucene toolkit for replicable information retrieval research
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.anserini.collection;
+
+import org.junit.Before;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+public class TwentynewsgroupsCollectiontest extends DocumentCollectionTest<TwentynewsgroupsCollection.Document> {
```

Comment:
`TwentynewsgroupsCollectiontest` should be `TwentynewsgroupsCollectionTest` for CI to trigger?

Reply:
Yes, that's why I can't see any test result. I got two tests failed though after changing it to the right name. I will push the code again after I fixed them. Thanks a lot!

Reply:
sg